### ISSUE TITLE
open files are getting increased due to NETTY configuration

### ISSUE THREAD

**Issue by deepraut89** [KostyaSha/yet-another-docker-plugin#123]
I am getting increase in no of "pipe" and "event-poll" type files after every save of Configure System page.
```
java     4658 jenkins 1064w     FIFO                0,8       0t0    550956 pipe
java     4658 jenkins 1065u  a_inode                0,9         0      5528 [eventpoll]
java     4658 jenkins 1066r     FIFO                0,8       0t0    550957 pipe
java     4658 jenkins 1067w     FIFO                0,8       0t0    550957 pipe
java     4658 jenkins 1068u  a_inode                0,9         0      5528 [eventpoll]
java     4658 jenkins 1069r     FIFO                0,8       0t0    550958 pipe
java     4658 jenkins 1070w     FIFO                0,8       0t0    550958 pipe
java     4658 jenkins 1071u  a_inode                0,9         0      5528 [eventpoll]
java     4658 jenkins 1072w     FIFO                0,8       0t0    550959 pipe
java     4658 jenkins 1073r     FIFO                0,8       0t0    550960 pipe
java     4658 jenkins 1074u  a_inode                0,9         0      5528 [eventpoll]
```
```
[jenkins@localhost ~]$ /usr/sbin/lsof -u jenkins  | grep -w 'pipe\|eventpoll]' | wc -l
128
[jenkins@localhost ~]$ /usr/sbin/lsof -u jenkins  | grep -w 'pipe\|eventpoll]' | wc -l
152
[jenkins@localhost ~]$ /usr/sbin/lsof -u jenkins  | grep -w 'pipe\|eventpoll]' | wc -l
176
[jenkins@localhost ~]$ /usr/sbin/lsof -u jenkins  | grep -w 'pipe\|eventpoll]' | wc -l
200
```
This increase is coming during the build when I changed connection type to NETTY and not with JERSEY.
Cloud configuration is same as I mentioned in Issue:119, but SSH as launch method.

**Comment by KostyaSha** [comment]
Is it socket connection?

**Comment by deepraut89** [comment]
It is showing as broken symlink in file descriptors:
```
ls /proc/4658/fd -l
lr-x------. 1 jenkins jenkins 64 Jan 18 19:07 940 -> pipe:[39684]
l-wx------. 1 jenkins jenkins 64 Jan 18 19:07 941 -> pipe:[39684]
lrwx------. 1 jenkins jenkins 64 Jan 18 19:07 942 -> anon_inode:[eventpoll]
lr-x------. 1 jenkins jenkins 64 Jan 18 19:07 943 -> pipe:[39685]
l-wx------. 1 jenkins jenkins 64 Jan 18 19:07 944 -> pipe:[39685]
lrwx------. 1 jenkins jenkins 64 Jan 18 19:07 945 -> anon_inode:[eventpoll]
lrwx------. 1 jenkins jenkins 64 Jan 18 19:07 946 -> socket:[39687]
lrwx------. 1 jenkins jenkins 64 Jan 18 19:07 947 -> socket:[37472]
[jenkins@localhost ~]$ file /proc/4658/fd/942
/proc/4658/fd/942: broken symbolic link to `anon_inode:[eventpoll]'
[jenkins@localhost ~]$ file /proc/4658/fd/941
/proc/4658/fd/941: broken symbolic link to `pipe:[39684]'
```

**Comment by KostyaSha** [comment]
What connection url do you use?

**Comment by KostyaSha** [comment]
@marcuslinke maybe you know?

**Comment by KostyaSha** [comment]
maybe something was fixed in docker-java 3.0.13, i updated master dependency

**Comment by KostyaSha** [comment]
please try latest version. Some issues were fixed in docker-java, but i not sure.
